target/linux/realtek/files-5.10/drivers/net/ethernet/rtl838x_eth.c

```diff
@@ -35,8 +35,8 @@ extern struct rtl83xx_soc_info soc_info;
  * available in total for all queues.
  */
 #define MAX_RXRINGS	32
-#define MAX_RXLEN	200
-#define MAX_ENTRIES	(200 * 8)
+#define MAX_RXLEN	300
+#define MAX_ENTRIES	(300 * 8)
```

Comment:
The commit message only says that you're changing 200 into 300, and will set FORCE_MAX_ZONEORDER, but not why you're doing that. Is this just making the DMA buffers bigger? Why does this setting need changing?

Reply:
FORCE_MAX_ZONEORDER allows to create larger DMA regions. The RX len and entries define the size of the DMA buffer, which gets set to the default used also in the SDK and gives a couple of percent in performance. If you do not change FORCE_MAX_ZONEORDER, then the allocation of that buffer will fail.

Reply:
Done.